modules/postgres/postgres_test.go

```diff
@@ -26,6 +29,54 @@ const (
 	password = "password"
 )
 
+func createSSLCerts(t *testing.T) (*tlscert.Certificate, *tlscert.Certificate, error) {
+	tmpDir := t.TempDir()
+	certsDir := tmpDir + "/certs"
+
+	if err := os.MkdirAll(certsDir, 0o755); err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
+		os.RemoveAll(tmpDir)
+	})
+
+	caCert := tlscert.SelfSignedFromRequest(tlscert.Request{
```

Comment:
Have you considered using tlscert.SelfSignedCA here instead? See https://github.com/mdelapenya/tlscert/blob/a038576e80ef259d2a60317ab51e37cdda8da3c8/tlscert.go#L135